source/administration/devops-command-center.rst

```diff
@@ -62,50 +62,124 @@ The set of steps contained within a playbook is called a checklist.
+7. Switch to the main channel view.
+8. Create an incident by selecting the playbook you just created.
 
-- Select the text field to edit the item details.
-- Hover over an input box and select the **X** to remove the checklist item.
-- Drag the hamburger menu to rearrange the checklist items.
+Editing checklists
+~~~~~~~~~~~~~~~~~~~
+
+You can edit checklist items and their position in the checklist when they are not part of an active incident.
 
-Choose **Done** to save the changes.
+1. Select **!** in the channel header of an active incident.
+2. Choose **Edit**.
+  - To edit: Select the text field to edit details.
+  - To delete: Hover over an input box and select the **X** to remove the checklist item.
+  - To move: Drag the hamburger menu to rearrange the checklist items.
+3. Choose **Done** to save the changes.
 
 Ending incidents
 ~~~~~~~~~~~~~~~~
 
 There are two ways to end an incident:
 
-- Using slash command */incident end* from within the incident channel.
+- Using the slash command */incident end* from within the incident channel.
 - Using the **End Incident** button in the RHS panel while in the incident channel.
 
-The incident will become inactive, be removed from the list of active incidents, and the associated channel will be archived.
+The incident will become inactive, is removed from the list of active incidents, and the associated channel is archived.
 
 Administrator's Guide
 ^^^^^^^^^^^^^^^^^^^^^^
 
+Incidents and playbooks are associated with teams in Mattermost. Participants can be added to an incident by other incident members inviting them to the incident's channel. Incidents are created based on playbooks, and are public or private depending on the playbook's settings. 
+
 Permissions
 ~~~~~~~~~~~~~~~~~~~~~
 
-Incidents and playbooks are associated with teams in Mattermost. Participants can be added by inviting them to the incident's channel.
+System Admins and Team Admins can edit the visibility of, and access to, playbooks and incidents so that:
+
+- Users who are removed from an incident channel by a System Admin can no longer access the incident's playbook.
+- Team members who aren't System or Team Admins can invite other team members to manage private playbooks.
+- Team members who aren't System or Team Admins can't be removed from an incident by other members of the team.
+- Team members who aren't System or Team Admins can't have access to playbooks removed by other members of the team.
+
+Managing incident channel visibility
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+System Admins can manage the visibility of channels by converting them into private incident channels. When a public incident is converted to a private incident, it can't be accessed by members who weren't in the channel at the time of the conversion, unless they're invited by a channel member. In addition, the contents of the channel won't show up in a search query or in the incident list in the incident backstage. 
```

Comment:
System admins, and I believe any channel administrator can do this as well. Most of this is "normal private channel behaviour", so wondering if we should eschew documenting that in favour of highlighting "what's different"?